### ISSUE TITLE
IOS  XCODE ERROR 

### ISSUE THREAD

**Issue by casa90** [Wizcorp/phonegap-facebook-plugin#1353]
Hello when i try to run xcode always says FacebookSDK.FacebookSDK.h not found so building process id failed. how can I solve this problem ?

**Comment by gulfcoastdev** [comment]
Same problem here. Did you manage to find a solution?

**Comment by casa90** [comment]
Yes I found. I used different plugin . Teleferik github was published working version.

**Comment by jehadja** [comment]
#1366 check out this i think you have same issue
